conda-forge.yml

```diff
@@ -2,7 +2,7 @@ conda_forge_output_validation: true
 provider: {linux_ppc64le: azure, linux_aarch64: azure}
 # remove the `azure` section and re-render before merging
 azure:
-  store_build_artifacts: False
+  store_build_artifacts: true
```

Comment:
Doh 🤦‍♂️

Thanks Leo 😅

Reply:
No worries! But I do notice one thing odd: I checked with the past record, it takes 40-50 mins to rerender this feedstock. Is this expected? Is it because this is a compiler package?

Reply:
Idk if I'd use the word "expected", but yes it does seem to take a long time to re-render this feedstock. Think that is a result of the rather large matrix of versions, architectures, etc. we include here. Issue ( https://github.com/conda-forge/conda-smithy/issues/1307 ) discusses long re-rendering cases and what options we might have to improve re-rendering time

Reply:
Thanks for pointer!